monai/handlers/stats_handler.py

```diff
@@ -74,7 +76,8 @@ def __init__(
         self.tag_name = tag_name
         self.key_var_format = key_var_format
         if logger_handler is not None:
-            self.logger.addHandler(logger_handler)  # pytype: disable=attribute-error
+            assert self.logger is not None
```

Comment:
line 74 should be: `self.logger = logging.getLogger(name)` and we can remove line 79 (because getLogger accepts None)

Reply:
@benjamin-gorman @crtrentz Can you please address these issues, and the "resolve" each request as they are completed?